app/routes.py

```diff
@@ -1,2 +1,344 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os
+import requests
+from dotenv import load_dotenv
+load_dotenv()
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+@tasks_bp.route("", methods=["POST", "GET"])
+def handle_tasks():
```

Comment:
Notice that the logic for GET and POST doesn't share any code, so if we wanted to try refactoring this we could consider putting the logic for each in separate functions, maybe `get_tasks` and `create_task`.